### PR TITLE
Apply media padding in Cupertino nav bar

### PR DESCRIPTION
**Pull request by cbracken**
Applies left and right media padding to Cupertino navigation bar large
titles.

### CODE REVIEW

**Comment by cbracken** [comment]
Fixes #13549

File: packages/flutter/test/cupertino/nav_bar_test.dart [hunk 1]
```diff
@@ -116,6 +116,58 @@ void main() {
...
+              );
+            },
+          );
+        },
+      ),
+    );
+
+    // Top padding is applied to middle.
+    expect(tester.getTopLeft(find.byKey(middleKey)), const Offset(277.5, 13.5 + 10.0));
+
```
Comment (goderbauer) [inline packages/flutter/test/cupertino/nav_bar_test.dart]: Is the double newline on purpose?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/test/cupertino/nav_bar_test.dart (hunk 1 above)
Comment (cbracken) [reply packages/flutter/test/cupertino/nav_bar_test.dart]: Nope! Fixed.

File: packages/flutter/test/cupertino/nav_bar_test.dart [hunk 2]
```diff
@@ -116,6 +116,58 @@ void main() {
...
+                ),
+              );
+            },
+          );
+        },
+      ),
+    );
+
+    // Top padding is applied to middle.
+    expect(tester.getTopLeft(find.byKey(middleKey)), const Offset(277.5, 13.5 + 10.0));
```
Comment (xster) [inline packages/flutter/test/cupertino/nav_bar_test.dart]: how did you get 13.5?

File: packages/flutter/test/cupertino/nav_bar_test.dart [hunk 3]
```diff
@@ -116,6 +116,57 @@ void main() {
...
+
+    // Top and left padding is applied to large title.
+    expect(tester.getTopLeft(find.byKey(titleKey)), const Offset(16.0 + 20.0, 58.0 + 10.0));
+  });
+
+
```
Comment (xster) [inline packages/flutter/test/cupertino/nav_bar_test.dart]: 2 lines here too

**Review by xster — APPROVED** [review]
LG

File: packages/flutter/test/cupertino/nav_bar_test.dart (hunk 2 above)
Comment (cbracken) [reply packages/flutter/test/cupertino/nav_bar_test.dart]: Measured with 0 padding.

File: packages/flutter/test/cupertino/nav_bar_test.dart (hunk 3 above)
Comment (cbracken) [reply packages/flutter/test/cupertino/nav_bar_test.dart]: Gah! My editor-fu has failed me twice today!

**Pull request merged**